ompi/mpi/c/isend.c

```diff
@@ -82,11 +82,11 @@ int MPI_Isend(const void *buf, int count, MPI_Datatype type, int dest,
 
     OPAL_CR_ENTER_LIBRARY();
 
+    rc = MCA_PML_CALL(isend(buf, count, type, dest, tag,
+                            MCA_PML_BASE_SEND_STANDARD, comm, request));
     MEMCHECKER (
         memchecker_call(&opal_memchecker_base_mem_noaccess, buf, count, type);
     );
```

Comment:
@ggouaillardet we should remove this code, it is incorrect based on today's MPI standard.